README.md

```diff
@@ -25,10 +25,23 @@ It’s a Rails app that:
 
    ```sh
    $ bundle install
+   ```
+
+5. Create development database 
+
+   ```sh
+   $ createdb webpage-versions-db_development
+   ```
+
+6. Edit the file ~\web-monitoring-db\config\database.yml. Uncomment line 35 and append your postgres database role password	
```

Comment:
@Mr0grog i installed Figaro in the app but i am still wondering how to take the password input from user. Should we ask them to edit some other file which does not contain any sensitive info or should we take input on the prompt while installation?